### DISCUSSION TITLE
Add new configuration files for FCOS3D MMDet 新 config 体验与适配 FCOS3D

### DISCUSSION THREAD

**Discussion by OpenMMLab-Assistant-004** [open-mmlab/OpenMMLabCamp#511] [MMDetection3D]
# Add new configuration files for FCOS3D 
 MMDet 新 config 体验与适配 FCOS3D 

According to the [Config adaptation document]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc))  and 
 the supported algorithm configurations, add all configurations 
 for FCOS3D and verify the correctness of inference and training.

mmengine 实现了一套新的 config 机制，基于新的 config 机制进行配置文件的适配
 - [相关适配教程文档]([https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)](https://aicarrier.feishu.cn/docx/PkICdaJbpoOVjLxMFi2cYqPjnOc)) 
 该任务适配的 config 目录：configs/fcos3d/ ，完成所有文件适配

## Task Information

**Repository**：MMDetection3D

**Difficulty**：Easy

**Points**：10

**Estimated Time Consumption**：14 days

## Task Status

**Claimant's Github ID**：0zzx0

**Claim Time**：2023年07月29日

**PR Link**：https://github.com/open-mmlab/mmdetection3d/pull/2679

## Points to Note

❗️请注意：此任务属于超级视客营任务，不能被领取！请登陆 [超级视客营平台](https://openmmlab.com/activity/codecamp) 按照流程申请任务，我们会尽快审核～

**Comment by 0zzx0** [comment]
https://github.com/open-mmlab/mmdetection3d/pull/2679
